### PR TITLE
"Quick tips for making a header"

### PR DESCRIPTION
**Pull request by indiegamehunt**

### CODE REVIEW

File: header-tips/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Quick tips for making a header
```
Comment (MaskofThoro) [inline header-tips/index.md:1]: Let's combine this with "What images can I use for a header" Doesn't make sense for it to have it's own section.

File: header-tips/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+If you don't want to make a custom image for your header, use a screenshot. A lot of screenshots turn out really nicely when blown up to header size.
+
+When taking your screenshot, run the game full-screen or in the highest resolution it supports. Make sure to capture the image as a JPG if it is pixel art, and as a PNG otherwise.
```
Comment (MaskofThoro) [inline header-tips/index.md:5]: Wouldn't you want to capture it as PNG if it is pixel art? Doesn't JPG make pixel art look blurry? Can we ask David/Nik/Yariv?

File: header-tips/index.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+### PNGs
```
Comment (MaskofThoro) [inline header-tips/index.md:7]: hmm I think the info here may need to be switched. This sounds more for JPG and the below sound more for PNG... can we look this up?

**Comment by indiegamehunt** [comment]
Addressing comments in edits and combining with #22 into #21

**Pull request closed without merging**